pyinfra/progress.py

```diff
@@ -24,8 +24,7 @@
 
 if IS_TTY:
     try:
-        from os import get_terminal_size
-        TERMINAL_WIDTH = get_terminal_size().columns
+        TERMINAL_WIDTH = os.get_terminal_size().columns
```

Comment:
The import is here because `os.get_terminal_size` doesn't exist in Python2! This should be more explicit though, as there's no explanation on that, will add a comment on that.

Reply:
Ah, but how does it affect this? Importing `from os import get_terminal_size` raises an exception in the same way as `import os` and calling the function as `os.get_terminal_size()` would in python2. I still think the global import `import os` should be avoided, but I think it's better than not having imports at the top.

See this:

```python
Python 2.7.17 (default, Nov  7 2019, 10:07:09) 
[GCC 7.4.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from os import get_terminal_size
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name get_terminal_size
>>> import os
>>> os.get_terminal_size()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'get_terminal_size'
```

Reply:
Agreed this is much nicer than the import; just needs the `ImportError` updated on L28 to `AttributeError`.

Reply:
Oh, I forgot that! Will fix, thanks! :+1: